src/components/App.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Display from './Display.js';
+import ButtonPanel from './ButtonPanel.js';
+
+function App() {
+  return (
+  <React.Fragment>
+    <div>
```

Comment:
- [ ] There's now a [short syntax](https://reactjs.org/docs/fragments.html#short-syntax) for declaring fragments. Also, we use fragments to minimize the use of divs. Kindly update
```
<>
   <Display />
   <ButtonPanel />
</>
```